src/app/containers/AppliancesPage.js

```diff
@@ -220,12 +225,14 @@ class AppliancesPage extends Component {
                     <li
                         className={`page-item ${this.state.pageStateUnconfigured ? "active" : ""}`}
                         onClick={this.changePageStateToUnconfigured}
+                        id="pageButtonUnconfigured"
```

Comment:
Test ids are recommended to start with component's name to be qniue through the application.
E.g. you can have "Apply" button for the several components.